engine/src/main/java/org/terasology/engine/subsystem/headless/renderer/HeadlessWorldRenderer.java

```diff
@@ -300,4 +295,15 @@ public int compare(RenderableChunk o1, RenderableChunk o2) {
         }
     }
 
+	@Override
+	public RenderingConfig getRenderingConfig() {
```

Comment:
for the OrthographicCamera (which calls super without parameters in its constructor) should I change it to call the new super(with worldProvider and renderingConfig as parameters) or just leave it like that? that camera is instanced in ShadowMapNode.java